src/utils/vagrant-template/setup.sh

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
```

Comment:
I think vagrant-template is not used anymore, we can delete it.

Reply:
I would prefer to do it in a follouwp PR to avoid mixing up too many things.